doc/guides/maintaining-npm.md

```diff
@@ -32,6 +32,7 @@ $ git reset --hard origin latest
 
```

Comment:
Unrelated but I think the last command on Step 1 should be `git reset --hard origin/latest`

Reply:
thanks, fixed